api-test/local.postman_environment.json

```diff
@@ -14,27 +14,27 @@
     },
     {
       "key": "tenant_id",
-      "value": "",
+      "value": "-",
```

Comment:
why need this ?

Reply:
otherwise k6 generates error due to empty string